plugins/azure/virtualnetworks/noNetworkGatewaysInUse.js

```diff
@@ -11,11 +11,19 @@ module.exports = {
     recommended_action: 'Configure subnets and network security groups instead of virtual network gateways',
     apis: ['resourceGroups:list','virtualNetworks:listAll','virtualNetworkGateways:listByResourceGroup'],
     realtime_triggers: ['microsoftnetwork:virtualnetworks:write','microsoftnetwork:virtualnetworks:delete','microsoftnetwork:virtualnetworkgateways:write','microsoftnetwork:virtualnetworkgateways:delete'],
-
+    settings: {
+        vNet_gatewayType: {
+            name: 'Virtual Network Gateway Type',
+            description: 'Gateway type that should not be configured with virtual networks i.e. Vpn, ExpressRoute. Enter "*" if no network gateways should be used irrespective of gateway type.',
+            regex: '^.*$',
```

Comment:
```suggestion
            regex: '^.*$',
```
regex should only allow vpn, expressroute and * as values